webapp/src/views/EventDetails.vue

```diff
@@ -2,45 +2,50 @@
   <div class="container">
     <router-link
       :to="{ path: `/event/${id}/register` }"
-      class="btn btn-primary mb-3 btnAn"
+      class="btn btn-outline-primary my-4 btnAn w-100"
       :class="{
         disabled: !freeSeats,
       }"
       >Person anmelden</router-link
     >
     <h1>Teilnehmer: {{ event?.name }}</h1>
-    <div v-for="program in event?.programs" :key="program.id">
-      <h3>{{ program.name }}</h3>
-      <table class="table table-striped">
-        <thead>
-          <tr>
-            <th scope="col">Vorname</th>
-            <th scope="col">Nachname</th>
-            <th scope="col">Abmelden</th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr v-for="participant in program.participants" :key="participant.id">
-            <td>{{ participant.givenName }}</td>
-            <td>{{ participant.familyName }}</td>
-            <th scope="row">
-              <button
-                type="button"
-                class="btn btn-outline-danger"
-                @click="
-                  removeParticipant(
-                    participant.id,
-                    participant.givenName,
-                    participant.familyName
-                  )
-                "
-              >
-                Abmelden
-              </button>
-            </th>
-          </tr>
-        </tbody>
-      </table>
+    <div v-for="program in event?.programs" :key="program.id" class="card my-4">
+      <div class="card-body">
+        <h3 class="card-title">{{ program.name }}</h3>
+        <table class="card-body w-100">
```

Comment:
Applying the class card-body for `<div>` and `<table>` leads to a too thick padding for mobile devices which causes unnecessary line breaks and does not look good.

Reply:
indeed , is removed